stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -497,44 +497,40 @@ extension TFState {
       targetNodeSpecs.append(helperFuncNode)
     }
 
-    if returnValues.count > 0 {
-      if _RuntimeConfig.executionMode.isTPU {
-        debugLog("Enable TPU execution.")
-        // When infeed is enabled, run it along with the output tensor nodes
-        // below.
-        let infeedEnqueueNode = TF_GraphOperationByName(graph,
-                                                        "InfeedEnqueueTuple")
-        if let infeedEnqueueNode = infeedEnqueueNode {
-          targetNodeSpecs.append(infeedEnqueueNode)
-          debugLog("Running enqueue with \(inputTensors.count) input tensors.")
-        }
+    if returnValues.count == 0 {
+        debugLog("""
```

Comment:
Un-indent?

Reply:
My emacs formats it for me this way, and it's also consistent with some debugLog code below, so I'll leave it as is.